db/structure.sql

```diff
@@ -2480,9 +2479,12 @@ ALTER TABLE ONLY public.winners
 SET search_path TO "$user", public;
 
 INSERT INTO "schema_migrations" (version) VALUES
+(20250220185334),
 (20250210211648),
+(20250205135218),
 (20250204131151),
 (20250202220815),
+(20250130144731),
```

Comment:
this PR should only add migration version `20250220185334`, these others are not present likely because they're from a different branch or your DB schema is out of sync but we can't add them here anyways

Reply:
```suggestion
(20250220185334),
(20250210211648),
(20250204131151),
(20250202220815),
```